package.json

```diff
@@ -67,6 +67,9 @@
     "tsx": "^3.14.0",
     "typescript": "^5.0.4"
   },
+  "resolutions": {
+    "simple-plist": "1.4.0-0"
+  },
```

Comment:
Is this resolution only relevant for bundling on our end? Asking because it has no effect when users install the wizard and I just wanna make sure that this is not necessary.

Reply:
This is actually bundled in from my fossilize work (#758). It causes a bundling issue and that should be the only change relevant to us. I'll double check the changes but the reports didn't mention anything notable.